### ISSUE TITLE
how could storage user info in redis ?

### ISSUE THREAD

**Issue by niechen2012** [auth0/java-jwt#339]
I see the code like this 
```java
 public static class Builder {
        private final Map<String, Object> payloadClaims;
        private Map<String, Object> headerClaims;

        Builder() {
            this.payloadClaims = new HashMap<>();
            this.headerClaims = new HashMap<>();
        }

```

when this used in distribute system , how can I do , I want to storage in redis 。how can I extension this

**Comment by lbalmaceda** [comment]
@niechen2012 I don't understand your question. Can you rephrase? What is it that you want to store?
